_schemaV2.graphql

```diff
@@ -10961,6 +10961,12 @@ type PriceInsightEdge {
   node: MarketPriceInsights
 }
 
+type PriceInsights {
```

Comment:
**(suggestion):** What do you think about naming it `ArtworkPriceInsights` because it's used for a sub-field of `Artwork`?